xrpl_api/src/types.rs

```diff
@@ -156,4 +156,13 @@ pub struct Transaction {
     pub meta: Option<Meta>,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+#[serde(tag = "LedgerEntryType")]
+pub enum LedgerEntry {
```

Comment:
Attempt to list all ledger objects such that they can be deserialized into this strongly types structure in responses